flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/JobGraph.java

```diff
@@ -652,4 +654,12 @@ public void setJobStatusHooks(List<JobStatusHook> hooks) {
     public List<JobStatusHook> getJobStatusHooks() {
         return this.jobStatusHooks;
     }
+
+    public long getClientHeartbeatTimeout() {
+        return clientHeartbeatTimeout;
+    }
+
+    public void setClientHeartbeatTimeout(long clientHeartbeatTimeout) {
+        this.clientHeartbeatTimeout = clientHeartbeatTimeout;
+    }
```

Comment:
1. I'd suggest to name this `get/setInitialClientHeartbeatTimeout`.
2. This might be more suitable to be put into `jobConfiguration`, rather than being a direct field of `JobGraph`.